scripts/benchmark_db/Gemfile

```diff
@@ -4,5 +4,14 @@ source 'https://rubygems.org'
 
 git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 
+gem 'concurrent-ruby', '~> 1.2', '>= 1.2.2'
+gem 'csv', '~> 3.2', '>= 3.2.6'
 gem 'deep_merge', '~> 1.2', '>= 1.2.2'
+gem 'fileutils', '~> 1.7'
+gem 'open3', '~> 0.1.2'
+gem 'optparse', '~> 0.3.1'
+gem 'pathname'
+gem 'set', '~> 1.0', '>= 1.0.3'
+gem 'tmpdir', '~> 0.1.3'
 gem 'toml-rb', '~> 2.2'
+gem 'tomlrb', '~> 2.0', '>= 2.0.3'
```

Comment:
Can you confirm that we need all of these dependencies? For example, `pathname` seems to be unused.